bitnami/mysql/templates/primary/statefulset.yaml

```diff
@@ -207,6 +207,21 @@ spec:
           {{- else if .Values.primary.customReadinessProbe }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.primary.customReadinessProbe "context" $) | nindent 12 }}
           {{- end }}
+          {{- if .Values.primary.startupProbe.enabled }}
```

Comment:
With the `customStartupProbe` is not working, could you fix it?

Reply:
@Mauraza I used exactly the same code as readiness... what do you mean by not working ?

Reply:
I missed change startupProbe.enabled to false, it's working